watch-parties/src/main/webapp/style.css

```diff
@@ -0,0 +1,48 @@
+body{
+  font-family: Roboto, Arial, sans-serif;
+}
+form{
+  display:flex;
+}
+.timePosted{
+  margin-left:108px;
+  margin-top:-26px;
```

Comment:
Negative margins are generally not advised. The layout for these video cells can be done pretty cleanly with [flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox). I threw an example together: https://jsfiddle.net/nLrwf1e0/2/
(We can chat about this if you want more info on flexbox)